ICSharpCode.Decompiler.Tests/ICSharpCode.Decompiler.Tests.csproj

```diff
@@ -1,129 +1,60 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <ProjectGuid>{FEC0DA52-C4A6-4710-BE36-B484A20C5E22}</ProjectGuid>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
-    <OutputType>Library</OutputType>
-    <RootNamespace>ICSharpCode.Decompiler.Tests</RootNamespace>
-    <AssemblyName>ICSharpCode.Decompiler.Tests</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <TargetFramework>net461</TargetFramework>
+
     <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
-    <NoStdLib>False</NoStdLib>
-    <WarningLevel>4</WarningLevel>
-    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
-    <CheckForOverflowUnderflow>False</CheckForOverflowUnderflow>
-    <NoWarn>67,169,1058,728,1720,649</NoWarn>
-    <TargetFrameworkProfile />
-    <NoWin32Manifest>False</NoWin32Manifest>
+
+    <NoWarn>$(NoWarn),67,169,1058,728,1720,649</NoWarn>
+
+    <GenerateAssemblyInfo>False</GenerateAssemblyInfo>
+
+    <EnableDefaultItems>false</EnableDefaultItems>
+
+    <OutputType>Exe</OutputType>
+    <StartupObject>ICSharpCode.Decompiler.Tests.Stub</StartupObject>
```

Comment:
Is this a workaround for the tooling bug with .NET Core Test Runner, that I mentioned in my rant?

Reply:
At this point, I'm not using the .NET Core test runner or the `dotnet` tool for anything. I don't expect that to change prior to converting Mono.Cecil.csproj to the new project system. The change in output type was actually a workaround to ensure `AutoGenerateBindingRedirects` properly created a config file, which is then used by the test runner. This fixes a few different problems related to assembly load failures during test runs without having to manually maintain the redirects in an **app.config** file.

Reply:
Thanks for explaining and answering my follow-up question about `AutoGenerateBindingRedirects` before I even posted it.